lib/model/rwfolder_test.go

```diff
@@ -463,3 +464,68 @@ func TestDeregisterOnFailInPull(t *testing.T) {
 		t.Fatal("Didn't get anything to the finisher")
 	}
 }
+
+func setupTestDataDir() {
+	os.RemoveAll("testdata23")
+	os.Mkdir("testdata23", 0755)
+	os.Chdir("testdata23")
+}
+
+func cleanupTestDataDir() {
+	os.Chdir("..")
+	os.RemoveAll("testdata23")
+}
```

Comment:
Not a fan of Chdir in tests to be honest.

I don't think it's hard to use longer paths in the rest of the test